src/main/resources/jenkins/plugins/extracolumns/UserNameColumn/config.jelly

```diff
@@ -23,11 +23,11 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:l="/lib/layout">
     <f:block>
         <tr>
             <td width="50">
-                <img src="${imagesURL}/48x48/user.png" title="${%User icon}" alt="${%User icon}" border="0"/>
+                <l:icon class="icon-person icon-xlg" title="${%User icon}" alt="${%User icon}"/>
```

Comment:
"icon-user" should be used instead of "icon-person".

Reply:
I've chosen the person icon here, because icon-user displays a group of people yet the title and description explicitly names a "User" where a single person fits better imo.

Reply:
You are right, semantically it's a better fit. It's too bad, that the person icon has just recently been added (Aug 24, 2021) so this will break older installations. :(

Reply:
Oh, I thought it used to be there for a longer duration.
See https://github.com/jenkinsci/extra-columns-plugin/pull/26